src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/domain/makerecalldecisions/recommendation/RecommendationTab.kt

```diff
@@ -0,0 +1,8 @@
+package uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.makerecalldecisions.recommendation
+
+data class RecommendationTab(
```

Comment:
It's minor, but the API doesn't need to know how it's presented, is `RecommendationsListItem` better?

Reply:
Updated - was in 2 minds what to call this. We have alot of similarly named classes.